src/game/piece.py

```diff
@@ -5,32 +5,38 @@ class Piece:
     - LightColor [True/False]
     - TopHole [True/False]"""
 
-    def __init__(self):
-        self.round_shape = False
-        self.big_size = False
-        self.light_color = False
-        self.top_hole = False
-        self.id = 1
+    def __init__(self, id=1, round_shape=False, big_size=False, light_color=False, top_hole=False):
+        self.round_shape = round_shape
+        self.big_size = big_size
+        self.light_color = light_color
+        self.top_hole = top_hole
+        self.id = id
+
+    def to_string(self):
+        return(1, ' : ', self.round_shape, ' - ', self.big_size, ' - ', self.light_color, ' - ', self.top_hole)
 
     @staticmethod
     def create_pieces_list():
         pieces_list = []
-        i = 0
-        while i < 16:
-            piece = Piece()
-            piece.id = i + 1
-            if i in [1, 3, 5, 9, 7, 11, 13, 15]:
-                piece.round_shape = True
-            if i in [2, 3, 6, 10, 7, 11, 14, 15]:
-                piece.big_size = True
-            if i in [4, 5, 6, 12, 7, 13, 14, 15]:
-                piece.light_color = True
-            if i in [8, 9, 10, 12, 11, 14, 13, 15]:
-                piece.top_hole = True
-            pieces_list.append(piece)
-            i += 1
+        pieces_list.append(Piece(1, False, False, False, False))
```

Comment:
You can do this directly, to avoid assignation operations:

return [
    Piece(1, False, False, False, False),
    Piece(1, False, False, False, False), 
    ....
]